index.html

```diff
@@ -1,315 +1,357 @@
-            </a>
-          </li>
-          <li class="social-icon">
-            <a href="https://github.com/ifnotlily/">
-              <img src="./images/github.svg" alt="Lilian's Github" />
-            </a>
-          </li>
-        </ul>
-      </div>
-      <div id="iteme-7"><button class="buttonresume" type="button">Get my Resume</button></div>
-    </div>
-    <div class="my-accordion">
-      <ul>
-        <li class='menu'>Languages
-
-
-        <li class='submenu'><a href=""><img src="./images/Ellipse1.png" alt="javascript">JavaSript</a></li>
-        <li class='submenu'><a href=""><img src="./images/Ellipse2.png" alt="html">HTML</a></li>
-        <li class='submenu'><a href=""><img src="./images/Ellipse3.png" alt="css">CSS</a>
-        </li>
-
-
-        <li class='menu'>Frameworks<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
-
-
-        <li class='menu'>Skills<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
-      </ul>
-
-    </div>
-
-  </section>
-  <section class="contactform">
-    <div class="container"></div>
-    <div class="content-form">
-      <h3>Contact me</h3>
-      <p>If you have an application you are interested in developing, a feature that you need built or a project that
-        needs coding. I’d love to help with it </p>
-      <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/xgerlrpz"
-        method="POST">
-        <fieldset id="fs-frm-inputs">
-          <input type="text" name="name" id="full-name" placeholder="Full name" maxlength="30" required>
-          <input type="email" name="_replyto" id="email-address" placeholder="Email address" required>
-          <textarea rows="5" name="message" id="message" placeholder="Enter text here" maxlength="500"
-            required></textarea>
-          <input type="hidden" name="_subject" id="email-subject" value="Contact Form Submission">
-        </fieldset>
-        <input type="submit" value="Get in touch" class="submit-button">
-      </form>
-
+      </section>
+      <section class="contact-section">
+        <form name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/xgerlrpz"
+          method="POST">
+          <h3>Contact me</h3>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+          <div class="coverUl">
+            <ul class="contact-form">
+              <li class="contact-item">
+                <input type="text" name="name" placeholder="Yeremias" maxlength="30" required/>
+              </li>
+              <li class="contact-item">
+                <input type="email" name="email" placeholder="natajayanj14@gmai.com" required/>
+              </li>
+              <li class="contact-item">
```

Comment:
[ OPTIONAL ] Text area section is not the same as the design. Please see the below screenshot and correct it accordingly. 
![text-box](https://user-images.githubusercontent.com/35504149/130118465-d6619a2d-b95b-4002-9bfb-4157a8fce21d.png)